Mathlib/Analysis/Normed/Lp/WithLp.lean

```diff
@@ -50,8 +51,19 @@ variable (p : ℝ≥0∞) (K : Type uK) (K' : Type uK') (V : Type uV)
 
 namespace WithLp
 
+variable {p V} in
+/-- The canonical equivalence between `V` and `WithLp p V`. This should always be used to convert
+back and forth between the representations. -/
+def toLp : V ≃ WithLp p V := Equiv.refl _
```

Comment:
nit:
```suggestion
def toLp : V ≃ WithLp p V := .refl _
```
Same below

Reply:
It's new in the sense that this syntax didn't exist in Lean 3, but Lean 4 by now isn't so new 😅

Reply:
I meant the style convention of *prefering* `.refl` to `Equiv.refl` :)